runtests/mpi/tester.py

```diff
@@ -96,7 +109,7 @@ def fixture(request):
 
     return fixture
 
-def MPITest(commsize):
+def MPITest(commsize, mpi_missing='fail'):
```

Comment:
I wonder if doing this is cleaner
```
@pytest.mark.skipif(MPI is None, "MPI is missing")
@MPITest([1, 2, 3, 4])
def test_bluh(comm):
  pass
```
Than plumbing the 'MPI is None' logic into MPITest.

Reply:
It's shorter but it doesn't achieve the same goals.

The main reason we added this feature is that when mpi_missing == "ignore",  the tests __not__ skipped but they are run with comm == None. This is because we tell our MPI libraries they are run in serial by providing comm=None as argument instead of `MPI.COMM_WORLD`.

Reply:
I see. Your application treat comm=None as MPI.COMM_SELF but without using mpi4py at all.

What about naming the argument mpi4py_missing_policy == "ignore". Probably also use _MPI4PY_MISSING instead _MPI4PY_UNSUPPORTED for the global flag for consistency. mpi_missing really sounds like a boolean to indicate whether MPI is missing. ;)

Reply:
Sounds good, renamed `mpi_missing` to `mpi_missing_policy` in e1175ff8e941f9769989005f6c3427de1a6b2fa1